packages/api-request-builder/src/create-service.js

```diff
@@ -42,7 +44,12 @@ export default function createService (
   if (!Array.isArray(definition.features) || !definition.features.length)
     throw new Error('Definition requires `features` to be a non empty array.')
 
+  if (!project.projectKey)
+    throw new Error('No project defined. Please enter a project key')
```

Comment:
Thought so too... But is there a scenario where ```createService``` can be called on it's own? (not in ```create-request-builder```)? Or I'll just do the validation here so it's throws no matter where it's called ;-)